plugin/advisor/advisor.go

```diff
@@ -192,6 +192,9 @@ const (
 	// PostgreSQLColumnRequirement is an advisor type for PostgreSQL column requirement.
 	PostgreSQLColumnRequirement Type = "bb.plugin.advisor.postgresql.column.require"
 
+	// PostgreSQLColumnCommentConvention is an advisor type for PostgreSQL column comment convention.
```

Comment:
Thanks for the contribution. Shall we have the rule support all types of objects other than columns as supported by PostgreSQL?

https://www.postgresql.org/docs/current/sql-comment.html

Reply:
 Sounds good, I'll ping you again when I'm done modifying

Reply:
Thanks, let me know whenever it's ready.